### PR TITLE
Fixes #202, Modified About, Using and Interpreting stack reports…

### LINKED ISSUE #202: Modify Stack Report Sections as per latest UX
- [x] Create Draft
- [x] Get Tech review <Sent for review, expecting answers to some questions raised during first round of discussions. Awaiting ack from SMEs>
- [ ] Create PR, get review and merge.

### PR DESCRIPTION
**Pull request by Preeticp**
…sections as per new UX.
Added screenshots, updated existing ones to enable reuse between guides. Also modified section on stack reports in GS guide.
@rkratky, please review and merge if it seems ok, it validates and builds fine. 
@mishaone, please note I have replaced one of the screenshots in the GS stack reports section, to enable reuse, also updated some content, and reformatted, based on the screenshot. Let me know if you feel there is anything amiss. 

BTW Misha, I just saw your question in the other issue, but did not dig deeper into the reference. I was referring to reusing images the way I have done here for:
detailed_stackreport.png or red_x.png
Here in case there is a context change in one guide and we need to modify these images, we might not be able to reuse it in the other guide, so does it make sense to rename such reused images as:
red_x_c.png[Error Icon] or c_red_x.png[Error Icon]? So that we are aware while modifying such images.

### CODE REVIEW

**Comment by Preeticp** [comment]
@rkratky, not sure what to make of the details provided here for the failing check: https://jenkins.cd.test.fabric8.io/blue/organizations/jenkins/fabric8io-auto-rel%2Ffabric8-online-docs/detail/PR-215/3/pipeline

File: docs/topics/modules/glossary.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[glossary]
+dependencies[glossary]
```
Comment (rkratky) [inline docs/topics/modules/glossary.adoc]: What's this?

File: docs/topics/modules/glossary.adoc [hunk 2]
```diff
@@ -59,11 +63,7 @@ iteration:: An iteration is used to organize, plan, and execute work items in a
 // endterm
 
 // term: f05a151a-61fa-45b1-8d8b-b3fd7bc63ea9, en_EN
-license conflict:: Licenses that conflict at the stack or component level.
-// endterm
-
-// term: 3a953b07-0cc3-4b45-b891-bf490216eae3, en_EN
```
Comment (rkratky) [inline docs/topics/modules/glossary.adoc]: You're removing a term (and thus an existing UUID) -- we cannot do that unless we have a confirmation from the Planner team that the UUID/term isn't consumed by them.

File: docs/topics/modules/interpreting_stack_report.adoc [hunk 3]
```diff
@@ -1,53 +1,48 @@
 [id="interpreting_stack_report"]
 = Interpreting your stack report
 
-The {osio} analytics report comprises three components:
+The Stack report provides a detailed analysis of your stack and it dependencies in the form of four cards.
```
Comment (rkratky) [inline docs/topics/modules/interpreting_stack_report.adoc]: Why capital "S"?

File: docs/topics/modules/interpreting_stack_report.adoc [hunk 4]
```diff
@@ -1,53 +1,48 @@
...
+* The *Usage outliers details* tab identifies and lists dependencies in your stack that are not commonly used in similar stacks or that do not work well with other dependencies in the stack. It suggests alternate dependencies, suitable to your stack, to replace them. The *Confidence score* depicts the confidence of {osio} Analytics on the suitability of the alternate dependency to your stack.
+* The *Companion component details* tab lists additional dependencies that you can add to your stack to enhance it. Based on the confidence score, you can decide on the suitability of the dependency to your stack and add it. You can also provide your feedback on the suggested dependencies.
 
-License:: The detailed analysis report lists licenses for each component, flags unknown and conflicting licenses, and recommends *Alternate Components* in case of *License Outliers*.
+== Dependency Details
+The stack report provides details of all the dependencies analyzed by it, and lists the unknown ones.
```
Comment (rkratky) [inline docs/topics/modules/interpreting_stack_report.adoc]: No comma.

File: docs/topics/modules/using_stack_report.adoc [hunk 5]
```diff
@@ -1,62 +1,62 @@
 [id="using_stack_report"]
 = Using your Stack Report
 
-{osio} Analytics helps you make informed choices with respect to your development stack and its component libraries by analyzing your stack at the development stage, within your IDE, and at the deployment stage.
+{osio} Analytics helps you make informed choices with respect to your development stack and its dependencies by analyzing your stack at the development stage, within your IDE, and at the deployment stage.
 
 == Development Stage
-{osio} Analytics analyzes your stack at the development stage and flags components with security vulnerabilities in your Che workspace. It also recommends a secure, alternate version for your stack.
+{osio} provides an integrated development environment in the form of Eclipse Che workspace to develop your code.
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: -> "an Eclipse Che workspace"

File: docs/topics/modules/using_stack_report.adoc [hunk 6]
```diff
@@ -1,62 +1,62 @@
...
+{osio} Analytics flags dependencies with security vulnerabilities by analyzing your stack, while you modify it, at the development stage within your Che workspace. It also suggests a secure, alternate version of the dependency for your stack.
 
-.Prerequisite
+.Prerequsite
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: It was correct before.

File: docs/topics/modules/using_stack_report.adoc [hunk 7]
```diff
@@ -1,62 +1,62 @@
...
 
-Ensure that you use link:e2e_workshop.html#creating_che_workspace-hello-world[Che workspace] to develop an application.
+. Create a link:e2e_workshop.html#creating_che_workspace-hello-world[Che workspace] to develop your stack.
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: The file `e2e_workshop.html` no longer exists. It's now `getting-started-guide.html`.

File: docs/topics/modules/using_stack_report.adoc [hunk 8]
```diff
@@ -1,62 +1,62 @@
...
-image::che_alt_recommendations.png[Stack Recommendations on Che]
+image::red_x.png[Error Icon]
 +
-. Update the component to the recommended version.
+. Update the dependencies to the suggested version.
+//looks like there is some change in the above step raised a bug, behavior on Che error now only flags CVE and does not recommend alternate version, this affects both GSG and UG.
 
 == Deployment Stage
-At the Deployment stage, {osio} Analytics analyzes your stack and provides a detailed stack analysis report. It analyzes various parameters of your stack components and recommends alternative and additional components suitable for your stack.
+When you create a new quickstart project, a new build/pipeline is executed. At this stage, OpenShift.io analyzes your stack and its dependencies and provides a detailed stack analysis report with insights on alternate and additional dependencies suitable for your stack.
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: Do not use "/" (slash) in doc text.

File: docs/topics/modules/using_stack_report.adoc [hunk 9]
```diff
@@ -1,62 +1,62 @@
...
 
 .Procedure
 
-* In the *Stack Report Recommendations* section of your space dashboard, click *Show complete stack report* to access the detailed stack report for your manifest files.
-
-Alternatively, you can access your stack report as follows:
-
-. Click *Create* and then *Pipelines* to view the pipeline builds for your project.
-. Click *Stack Report* to see the stack and component level analyses for your project.
+. Click *Create*, and then *Pipelines* to view the pipeline builds for your project.
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: No comma.

File: docs/topics/modules/using_stack_report.adoc [hunk 10]
```diff
@@ -1,62 +1,62 @@
...
+. In the pipelines view, click btn:[Stack Report] to see the analysis report for your entire stack.
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: -> `*Pipelines* view`

File: docs/topics/modules/using_stack_report.adoc [hunk 11]
```diff
@@ -1,62 +1,62 @@
...
-* *Additional components recommended by {osio}:* A list of additional components commonly used by similar stacks.
+image::pipeline_stackreport.png[Acessing Stack Report]
++
+The report displays a summary of four key aspects relevant to your stack in the form of cards:
++
+image::full_stack_reports.png[Full Stack Report]
 +
-See <<interpreting_stack_report>> section for further details.
+Security Issues:: This card highlights the number of security issues in your stack, the highest Common Vulnerability Scoring System (CVSS) score, and the number of dependencies with this high score.
+Licenses:: This card suggests an appropriate stack level license, and flags conflicting, unknown, and restrictive licenses (licenses that are not commonly used in similar stacks or that do not work well with the stack’s representative license) affecting your stack.
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: No first comma.

File: docs/topics/modules/using_stack_report.adoc [hunk 12]
```diff
@@ -1,62 +1,62 @@
...
+Insights:: Based on the analysis of other similar stacks, this card identifies usage outliers (dependencies not commonly used in similar stacks and that rarely go well together) in your stack and also highlights the number of companion (additional) dependencies that could augment your stack.
+Dependency Details:: This card lists the number of dependencies analyzed by {osio} and those unknown to it.
+. Click each of the cards to see <<interpreting_stack_report,detailed information>> about each of the aspects relevant to your stack.
+. Expand the arrow adjacent to the dependency to see information about:
 
-. Filter the *Detail analysis of your stack components* list to see:
+* The current and latest available version
+* Licenses, and tags associated with it
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: No comma.

File: docs/topics/modules/using_stack_report.adoc [hunk 13]
```diff
@@ -1,62 +1,62 @@
...
-** License Outliers (image:license_outlier.png[title="License Outliers"])
-** Unknown Licenses (image:license_unknown.png[title="Unknown Licenses"])
-** License conflict in a Component (image:license_conflicts.png[title="License Conflicts"])
 +
-Note that {osio} recommends *Alternate Components* for your stack if it identifies license or usage outliers.
+These statistics help you compare the existing dependency with the alternate dependency and make a smart choice for your stack.
 
-. In the *Action* column adjacent to the recommended alternate or additional components, click *Create Work Item* to create a work item in the {osio} Planner and <<working_with_an_existing_work_item,follow through>> on the recommendations.
+. To act on the analytics and insights provided by the report:
+* In the Security Issues detailed view, click *Report an issue* to report a security vulnerability highlighted in the report as an issue in the {osio} planner.
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: -> `*Security Issues* detailed view`

-> Planner?

File: docs/topics/modules/using_stack_report.adoc [hunk 14]
```diff
@@ -1,62 +1,62 @@
...
+* In the Insights detailed view, click *Create work item* to create issues for adding the suggested alternate or companion dependency. The issue is created in planner and auto populated with the information about the suggested dependency.
```
Comment (rkratky) [inline docs/topics/modules/using_stack_report.adoc]: -> *Insights* detailed view`

-> Planner?

-> auto-populated

**Comment by rkratky** [comment]
> not sure what to make of the details provided here for the failing check

It's the deployment step failing. I.e. docs are building fine (see the steps visualized at the top of the page -- click on the "Details" link), but the container with the build isn't deployed to OSO. Bug filed at https://github.com/fabric8-apps/fabric8-online-docs-app/issues/29

File: docs/topics/modules/glossary.adoc (hunk 2 above)
Comment (Preeticp) [reply docs/topics/modules/glossary.adoc]: As per our discussion on this, I have used this UUID for the term Restrictive license. As per latest Analytics UX, License outlier is replaced with Restrictive license. Planner team will not consume this as it is part of Analytics UX.

File: docs/topics/modules/using_stack_report.adoc (hunk 13 above)
Comment (Preeticp) [reply docs/topics/modules/using_stack_report.adoc]: Leaving 'planner' as is, cause Planner is not used in the UX, similarly analytics is not used anywhere, so referring to it as {osio} analytics.

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-215-5 is available for testing at http://fabric8-online-docs-app-pr-215-fabric8-online-docs.badger.fabric8.io

**Pull request merged**